### PR TITLE
Remove deprecated sqlite modules

### PR DESCRIPTION
**Pull request by astafan8**

### CODE REVIEW

File: docs/examples/driver_examples/Qcodes example with keithley 2450.ipynb [hunk 1]
```diff
@@ -17,7 +17,7 @@
    "source": [
     "import qcodes as qc \n",
     "from qcodes.instrument_drivers.tektronix.Keithley_2450 import Keithley2450\n",
-    "from qcodes.dataset.database import initialise_database\n",
+    "from qcodes.dataset.sqlite.database import initialise_database\n",
```
Comment (jenshnielsen) [inline docs/examples/driver_examples/Qcodes example with keithley 2450.ipynb:20]: We should eventually import all these from `qcodes.dataset`

Comment (astafan8) [reply docs/examples/driver_examples/Qcodes example with keithley 2450.ipynb:20]: i can do that now :)

**Comment by jenshnielsen** [comment]
@astafan8 can you have a quick look that my conflict resolition is as you want it

**Comment by astafan8** [comment]
@jenshnielsen it looks ok, didn't notice anything wrong

**Pull request merged**